husky_components/matrix/main.js

```diff
@@ -65,12 +66,12 @@ define(function() {
         },
 
         toggleRow: function(event) {
-            var $tr = this.sandbox.dom.parent(event.currentTarget),
+            var $tr = this.sandbox.dom.parent(this.sandbox.dom.parent(event.currentTarget)),
                 $targets = this.sandbox.dom.find('span[class^="fa-"]', $tr),
                 $activeTargets = this.sandbox.dom.find('span[class^="fa-"].' + activeClass, $tr),
-                $link = this.sandbox.dom.find('td:last-child span', $tr), activated;
+                $link = this.sandbox.dom.find('td:last-child > span:last-of-type', $tr), activated;
```

Comment:
This selector is already used in my previous variable. How about extracting it to a separate variable?